packages/evm-contracts/contracts/IValist.sol

```diff
@@ -150,6 +162,16 @@ interface IValist {
   /// @param _address Address of member.
   function removeTeamMember(string memory _teamName, address _address) external;
 
+  /// Set team beneficiary address for recieving payments.
+  ///
+  /// @param _teamID Unique ID of the team.
+  /// @param _newBeneficiary Address of new beneficiary address.
+  function setBeneficiary(
```

Comment:
`setTeamBeneficiary`would be more consistent.

Reply:
updated!